Fedoraware/TeamFortress2/TeamFortress2/Features/Auto/AutoBlast/AutoBlast.cpp

```diff
@@ -87,6 +87,30 @@ void CAutoAirblast::Run(CBaseEntity* pLocal, CBaseCombatWeapon* pWeapon, CUserCm
 			}
 		}
 
+		if (Vars::Triggerbot::Blast::ExtinguishPlayers.Value)
+		{
+			for (const auto& pBurningPlayer : g_EntityCache.GetGroup(EGroupType::PLAYERS_TEAMMATES))
+			{
+				Vec3 vPredicted = (pBurningPlayer->GetAbsOrigin() + pBurningPlayer->GetVelocity().Scale(flLatency / 1000.f));
```

Comment:
Is it really necessary to predict the enemy position?

Reply:
No